docs/connectors/postgresql.asciidoc

```diff
@@ -733,6 +733,37 @@ Other than PostgreSQL's `TIMESTAMPTZ` and `TIMETZ` data types (which contain tim
 
 |=======================
 
+When `time.precision.mode` configuration property is set to `adaptive_time_microseconds`, then the connector will determine the literal type and semantic type for the temporal types based on the column's data type definition so that events _exactly_ represents the values in the database, except that all TIME fields will be captured as microseconds:
```

Comment:
s/When/When the/
s/represents/represent/

Reply:
done